crates/matrix-sdk-ui/src/timeline/mod.rs

```diff
@@ -604,6 +604,7 @@ impl Timeline {
     ///   state of `SendState::NotYetSent` might be supported in the future as
     ///   well, but there can be no guarantee for that actually stopping the
     ///   event from reaching the server.
+    #[instrument(skip_all)]
     pub async fn cancel_send(&self, txn_id: &TransactionId) -> bool {
```

Comment:
Do we want to skip the transaction id? it's logged in other places, so having it here could help identifying what's been cancelled.

Reply:
Yeah you're right, should be logged.